pkg/daemon/update.go

```diff
@@ -1469,12 +1469,22 @@ func (dn *Daemon) writeUnits(units []ign3types.Unit) error {
 	for _, u := range units {
 		// write the dropin to disk
 		for i := range u.Dropins {
+			dpath := filepath.Join(pathSystemd, u.Name+".d", u.Dropins[i].Name)
+
 			if u.Dropins[i].Contents == nil || *u.Dropins[i].Contents == "" {
 				glog.Infof("Dropin for %s has no content, skipping write", u.Dropins[i].Name)
```

Comment:
Can you update this to mention that we are going to try to remove a file? That will give us some more context if the remove operation fails.

Reply:
Sure